ADApp/pluginSrc/NDPluginDriver.cpp

```diff
@@ -72,6 +72,10 @@ static const char *driverName="NDPluginDriver";
     if (dimsChanged) {
         doCallbacksInt32Array(this->dimsPrev, ND_ARRAY_MAX_DIMS, NDDimensions, 0);
     }
+    // Save a pointer to the input array for use by ProcessPlugin
+    if (pInputArray_) pInputArray_->release();
+    pInputArray_ = pArray;
+    pArray->reserve();
```

Comment:
I don't think there was a danger the other way, since the pArray has at least one reference count, but I've changed since it looks better.